mlflow/store/tracking/rest_store.py

```diff
@@ -183,6 +185,9 @@ def create_run(self, experiment_id, user_id, start_time, tags):
 
         :return: The created Run object
         """
+        if MLFLOW_RUN_NAME not in [tag.key for tag in tags]:
+            tags.append(RunTag(MLFLOW_RUN_NAME, _generate_random_name()))
+
```

Comment:
Rather than adding this in rest_store, which is still a client-side abstraction, can we add the run name generation to the `mlflow server` handler here: https://github.com/mlflow/mlflow/blob/7219771141c7140fd68f6ed3a5b649e5ad1dab13/mlflow/server/handlers.py#L666-L683 ?

Reply:
added (and removed from rest_store)!